src/install/oracle/users.mdx

```diff
@@ -0,0 +1,113 @@
+---
+componentType: default
+headingText: Create Oracle database users
+---
+
+You now need to create an Oracle database user with assigned privileges. Follow the instructions below. Be sure to replace values like 'USERNAME' with your information.
```

Comment:
I'm not sure if our docs is the place to explain this, but maybe linking out to Oracle docs that explain why this is necessary. I'm not convinced tho. On one hand, it's nice not to hunt for docs about an action you have to do if you're unfamiliar with it; the other hand, you're sending people out of the ~flow)